internal/routing.go

```diff
@@ -90,14 +98,46 @@ func (r *routingHandler) sendPacket(p types.Packet, src types.NodeAddress) error
 		go r.notifyDecision(p, src, r.pk.Hash())
 		return nil
 	}
-	next, err := r.reachability.FindNextHop(p.Destination(), src)
+	possible_next, err := r.reachability.FindNextHop(p.Destination(), src)
```

Comment:
It seems like all of this should be encapsulated within the bandwidth module. Maybe a better way to organize this is

reachability module, which provides information about possible paths based on topology.
bandwidth module, which provides information about current bandwidth + historical bandwidth to various nodes.
Some bandwidth based routing module, which consumes both data sources and does what you're doing here about picking the best.
A super simple routing module which just selects the first next hop from the routing module.

That way we can compare both of them and actually make graphs about which one is better.